src/components/stats.js

```diff
@@ -0,0 +1,159 @@
+                    <div className="stat">
+                        <span className="statAmount">123,456</span>
+                        <span className="statName">Stat Name</span>
+                    </div>
+                    <div className="stat">
+                        <span className="statAmount">123,456</span>
+                        <span className="statName">Stat Name</span>
+                    </div>
+                    <div className="stat">
+                        <span className="statAmount">123,456</span>
+                        <span className="statName">Stat Name</span>
+                    </div>
+                </div>
+                <div id="defense" className="statSection">
+                    <h6>Defense</h6>
+                    <div className="stat">
+                        <span className="statAmount">123,456</span>
+                        <span className="statName">Stat Name</span>
+                    </div>
+                    <div className="stat">
+                        <span className="statAmount">123,456</span>
+                        <span className="statName">Stat Name</span>
+                    </div>
+                    <div className="stat">
+                        <span className="statAmount">123,456</span>
+                        <span className="statName">Stat Name</span>
+                    </div>
+                </div>
+                <div id="ss1" className="statSection">
+                    <h6>Skill Slot 1</h6>
+                    <div className="stat">
+                        <span className="statAmount">123,456</span>
+                        <span className="statName">Stat Name</span>
+                    </div>
+                    <div className="stat">
+                        <span className="statAmount">123,456</span>
+                        <span className="statName">Stat Name</span>
+                    </div>
+                    <div className="stat">
+                        <span className="statAmount">123,456</span>
+                        <span className="statName">Stat Name</span>
+                    </div>
+                </div>
+                <div id="ss2" className="statSection">
+                    <h6>Skill Slot 2</h6>
+                    <div className="stat">
+                        <span className="statAmount">123,456</span>
+                        <span className="statName">Stat Name</span>
+                    </div>
+                    <div className="stat">
+                        <span className="statAmount">123,456</span>
+                        <span className="statName">Stat Name</span>
+                    </div>
+                    <div className="stat">
+                        <span className="statAmount">123,456</span>
+                        <span className="statName">Stat Name</span>
+                    </div>
+                </div>
+            </div>
+            <div className="grid statBreakdownGrid">
```

Comment:
Should this all be one component? It seems like this is all concerning the same thing.

Reply:
I made this a separate component in the most recent commit